<launch>
  <!-- Arguments -->
  <arg name="open_rviz" default="true"/>
  <arg name="move_forward_only" default="false"/>
  <arg name="scan_topic" default="scan_filtered"/>

  <!-- Wheelchair bringup -->
  <include file="$(find wheelchair_bringup)/launch/wheelchair_remote.launch">
  </include>

  <!-- Move_base -->
  <include file="$(find wheelchair_door)/launch/door_move_base.launch">
    <arg name="move_forward_only" value="$(arg move_forward_only)"/>
  </include>

  <!-- Go to door 
  <include file="$(find wheelchair_door)/launch/gotodoor.launch">
  </include>
  -->

  <!-- Rviz  -->
  <group if="$(arg open_rviz)"> 
    <node pkg="rviz" type="rviz" name="rviz" required="true"
          args="-d $(find wheelchair_navigation)/rviz/door_navi.rviz"/>
  </group>
 
</launch>
